pkg/apis/pipeline/v1alpha2/taskrun_types.go

```diff
@@ -0,0 +1,302 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
```

Comment:
one thing thats not clear to me reviewing: how many of these files and tests are duplicated from v1alpha1 and how many are new? (maybe we can include a docstring or something in the copied files explaining that, maybe in the v1lapha1 version, something like "this file is frozen, changes should now happen in <path to v1alpha2>" <-- we could even add automation if we do that to make sure ppl aren't changing files they shouldnt)

Reply:
bumpin this :innocent: